packages/node/src/evm/authorization/authorization-fetching.ts

```diff
@@ -109,6 +109,14 @@ export async function fetch(
     return [[], {}];
   }
 
+  // If there are no authorizer contracts then endpoint is public
+  if (isEmpty(fetchOptions.authorizers)) {
```

Comment:
Nice and simple 👍🏻

Reply:
Actually a nice trick to convey the same information is to create a very specific function/variable for that:
```js
const isEndpointPublic = isEmpty(fetchOptions.authorizers)
if (isEndpointPublic) ...
```

Ofc, this is perfectly fine - but I've read this tip from a smartass from some book :) :D

Reply:
I really like that idea but I think I like consistency more tbh 😄